cli/src/commit_templater.rs

```diff
@@ -251,6 +251,22 @@ impl CommitTemplateBuildFnTable<'_> {
             shortest_id_prefix_methods: builtin_shortest_id_prefix_methods(),
         }
     }
+
+    pub fn new() -> Self {
+        CommitTemplateBuildFnTable {
+            core: CoreTemplateBuildFnTable::new(),
+            commit_methods: HashMap::new(),
+            ref_name_methods: HashMap::new(),
+            commit_or_change_id_methods: HashMap::new(),
+            shortest_id_prefix_methods: HashMap::new(),
+        }
+    }
+}
+
+impl<'a> Default for CommitTemplateBuildFnTable<'a> {
```

Comment:
Perhaps, `#[derive(Default)]` will work here.

Reply:
Went with `empty()` for consistency